service/controller/v2/arm_templates/vmss.json

```diff
@@ -189,9 +189,7 @@
                 "tier": "Standard",
                 "capacity": "[int(parameters('vmssVmCount'))]"
             },
-            "identity": {
-                "type": "systemAssigned"
-            },
+            "identity": "[if(parameters('vmssMSIEnabled'), json('{\"type\":\"systemAssigned\"}'), json('{\"type\":\"None\"}'))]",
```

Comment:
In this case I'd like to simplify it to:

```json
"identity": {
    "type": "[if(parameters('vmssMSIEnabled'), 'systemAssigned', 'None')]"
},
```

Reply:
Looks better, right